client/src/featureform/tls.py

```diff
@@ -0,0 +1,15 @@
+import grpc
+import os
+
+def insecure_channel(host):
+    return grpc.insecure_channel(host, options=(('grpc.enable_http_proxy', 0),))
+
+def secure_channel(host, cert_path):
+    cert_path = cert_path or os.getenv('FEATUREFORM_CERT')
```

Comment:
if both are set, do we want to use `cert_path` or `FEATUREFORM_CERT`?

Reply:
We want cert_path because it allows an override over the env var